### PR TITLE
Fixes some grammar checks.

### PR DESCRIPTION
**Pull request by Naaanii**
## About The Pull Request
Title
## Why It's Good For The Game
Fix and consistency good.
## Changelog
:cl:
spellcheck: added improper tag to some guns missing it.
/:cl:

### CODE REVIEW

File: code/game/objects/machinery/vending/loadout_vendor.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /obj/machinery/loadout_vendor
-	name = "automated vendor"
+	name = "\improper Automated Loadout Vendor"
```
Comment (TiviPlus) [inline code/game/objects/machinery/vending/loadout_vendor.dm:2]: Huh? but this isnt a proper name it should be lowercase

File: code/modules/projectiles/guns/shotguns.dm [hunk 2]
```diff
@@ -560,7 +560,7 @@
 // Leicester Rifle. The gun that won the west.
 
 /obj/item/weapon/gun/shotgun/pump/lever/repeater
-	name = "Leicester Repeater"
+	name = "\improper Leicester Repeater"
```
Comment (TiviPlus) [inline code/modules/projectiles/guns/shotguns.dm:563]: shouldnt this just be lowercase

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Rest lgtm

File: code/game/objects/machinery/vending/loadout_vendor.dm (hunk 1 above)
Comment (Naaanii) [reply code/game/objects/machinery/vending/loadout_vendor.dm:2]: Yeah, but weapons vendor follow the same format despite not being proper nouns.
Should I change that instead?

File: code/modules/projectiles/guns/shotguns.dm (hunk 2 above)
Comment (Naaanii) [reply code/modules/projectiles/guns/shotguns.dm:563]: Proper noun

File: code/game/objects/machinery/vending/loadout_vendor.dm (hunk 1 above)
Comment (TiviPlus) [reply code/game/objects/machinery/vending/loadout_vendor.dm:2]: Yea

Comment (Naaanii) [reply code/game/objects/machinery/vending/loadout_vendor.dm]: Done

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**